packages/async-rewriter/package.json

```diff
@@ -4,8 +4,8 @@
   "description": "MongoDB Shell Async Rewriter Package",
   "main": "./lib/index.js",
   "scripts": {
-    "test": "mocha --timeout 15000 --colors -r ts-node/register \"./{src,lib}/**/*.spec.ts\"",
-    "test-ci": "mocha --timeout 15000 -r ts-node/register \"./{src,lib}/**/*.spec.ts\"",
+    "test": "mocha --timeout 60000 --colors -r ts-node/register \"./{src,lib}/**/*.spec.ts\"",
```

Comment:
do we still need this large of a timeout? or was that just for the azure pipelines windows tests?

Reply:
Im not sure - @mcasimir ?

Reply:
That aligns with what was inside the `mocha` tests: `this.timeout(60000)` was overriding the 15000 from the command line. 

In general 1 minute is not a large timeout for the CI, and it won't make the tests take longer in case everything completes before, is not bad being a bit tolerant if the machines are busy.

A nice thing that we could do is see if we can override it with an environment variable, so that we can tweak it everywhere without a commit and have a more 'fail-fast' value locally.

But honestly unless it would cause an issue that should be a sensible value.